cocos/core/assets/asset.ts

```diff
@@ -118,9 +118,6 @@ export class Asset extends Eventify(RawAsset) {
      * @type {String}
      * @readOnly
      */
-    @property({
-        visible: false,
-    })
     get nativeUrl () {
```

Comment:
This shouldn't be modified, what's the purpose ?

Reply:
> This shouldn't be modified, what's the purpose ?

This property only have a getter. There is no corresponding attribute that needs to be serialized, and the editor encounters an error when modifying the data.